pkg/crypto/keys/private_key_test.go

```diff
@@ -14,10 +14,6 @@ import (
 func TestPrivateKey(t *testing.T) {
 	for _, testCase := range keytestcases.Arr {
 		privKey, err := NewPrivateKeyFromHex(testCase.PrivateKey)
-		if testCase.Invalid {
```

Comment:
We still have invalid cases and we still need to test them, so why this is removed?